.eslintrc.cjs

```diff
@@ -3,14 +3,10 @@ module.exports = {
     browser: true,
     es2021: true,
   },
-  extends: [
-    'airbnb-base',
-    'prettier',
-  ],
+  extends: ["airbnb-base", "prettier"],
```

Comment:
Hmm unintentional?

Reply:
eh? I don't remember changing this. Could be linter?

Reply:
Might want to revert the changes in this file!